frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeaturesColumnsMenu/ProjectFeaturesColumnsMenu.tsx

```diff
@@ -12,47 +13,49 @@ export const ProjectFeaturesColumnsMenu: FC<
     ProjectFeaturesColumnsMenuProps
 > = ({ columnVisibility, environments, onToggle }) => {
     return (
-        <ColumnsMenu
-            columns={[
-                {
-                    header: 'Name',
-                    id: 'name',
-                    isVisible: columnVisibility.name,
-                    isStatic: true,
-                },
-                {
-                    header: 'Created',
-                    id: 'createdAt',
-                    isVisible: columnVisibility.createdAt,
-                },
-                {
-                    header: 'By',
-                    id: 'createdBy',
-                    isVisible: columnVisibility.createdBy,
-                },
-                {
-                    header: 'Last seen',
-                    id: 'lastSeenAt',
-                    isVisible: columnVisibility.lastSeenAt,
-                },
-                {
-                    header: 'Lifecycle',
-                    id: 'lifecycle',
-                    isVisible: columnVisibility.lifecycle,
-                },
-                {
-                    id: 'divider',
-                },
-                ...environments.map((environment) => ({
-                    header: environment,
-                    id: formatEnvironmentColumnId(environment),
-                    isVisible:
-                        columnVisibility[
-                            formatEnvironmentColumnId(environment)
-                        ],
-                })),
-            ]}
-            onToggle={onToggle}
-        />
+        <Box sx={(theme) => ({ marginLeft: theme.spacing(1) })}>
+            <ColumnsMenu
```

Comment:
What is this component? And is there any reason why you couldn't apply the margin on the `ColumnsMenu` directly? Not important, but I'm just not sure what the hierarchy is here.